NewSpring/ApollosProject
# (DEV-3736) - Adds smooth animation to the add prayer card and menu

## DESCRIPTION

### What does this PR do, or why is it needed?
Adds animation to add prayer card and menu. This adds the polish needed for a professional application.

### How do I test this PR?
1. Open up the app
2. Make sure animations are working as intended.
3. Check tests.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
